Content.Server/Light/EntitySystems/ExpendableLightSystem.cs

```diff
@@ -134,8 +135,8 @@ private void UpdateVisualizer(Entity<ExpendableLightComponent> ent, AppearanceCo
 
                 case ExpendableLightState.Dead:
                     _appearance.SetData(ent, ExpendableLightVisuals.Behavior, string.Empty, appearance);
-                    var isHotEvent = new IsHotEvent() {IsHot = true};
-                    RaiseLocalEvent(ent, isHotEvent);
+                    var ignite = new IgnitionEvent() {Ignite = true};
```

Comment:
Is this supposed to be false, since it's dead?

Reply:
It was still hot before, I thought it was because it was like a burnt husk and still warm like ash? It was probably a mistake, I can fix that.